### ISSUE TITLE
Remove the need of lambdaws.start

### ISSUE THREAD

**Issue by EFF** [mentum/lambdaws#18]

**Comment by ghostsnstuff** [comment]
@EFF agreed - seems redundant.

**Comment by ghostsnstuff** [comment]
@EFF in reference to 

```exports.start = function (startedCallback) {
    // TODO : move this in the function invocation ?
    _sqsHelper.startQueue(startedCallback);
};```

 Which invocation are you referring to? `_initializeAwsHelpers`? `SQSHelper`? Or `LambdaHelper`?

**Comment by EFF** [comment]
@ghostsnstuff I mean when the clouded function is invoked.
When you 
``` 
var cloudedFunc = lamdaws.create(myfunc);
// and then 
cloudedFunc(); // this is the function invocation to me
```
I haven't look for better solutions that's the thing that came to me, there might be better solutions.
You're welcome to assign yourself in the [public trello board](https://trello.com/b/V8OrXkFa/lambda)

**Comment by ghostsnstuff** [comment]
@EFF gotcha, so from the user's perspective. I wasn't sure if you were referring to some internal optimization. I'll take a look again after work tonight.

**Comment by EFF** [comment]
@ghostsnstuff I wasn't super clear my bad !

**Comment by ghostsnstuff** [comment]
@EFF no stress man - sounds good :+1:

**Comment by EFF** [comment]
Fixed in #54
